src/openfermion/ops/__init__.py

```diff
@@ -10,6 +10,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+from ._symbolic_operator import SymbolicOperator
```

Comment:
I suppose it is necessary to import this before FermionOperator? If so then you get a pass on the alphabetical imports...

Reply:
Yup, exactly.